src/Public/Responses.ps1

```diff
@@ -19,14 +23,23 @@ Set-PodeResponseAttachment -Path './image.png'
 
 .EXAMPLE
 Set-PodeResponseAttachment -Path 'c:/content/accounts.xlsx'
+
+.EXAMPLE
+Set-PodeResponseAttachment -Path './data.txt' -ContentType 'application/json'
+
+This command specifies that the content stored in data.txt should be considered as JSON data rather than plain text.
```

Comment:
The documentation for Pode's function is auto-generated using PlatyPS, and at the moment, it doesn't play too well with multi-line examples. I'll add this into the contributing file, but just the first line with the code example is all right, I'd remove the descriptive one - so long as the new parameter's description is good enough (which it is 👍).